cmd/security-agent/subcommands/runtime/command_linux.go

```diff
@@ -8,56 +8,43 @@
 package runtime
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/spf13/cobra"
 
 	"github.com/DataDog/datadog-agent/cmd/security-agent/command"
-	"github.com/DataDog/datadog-agent/pkg/security/proto/api"
+	sysprobecmd "github.com/DataDog/datadog-agent/cmd/system-probe/command"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/subcommands/runtime"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/subcommands/runtime/policy"
 )
 
 // Commands returns the config commands
 func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 	runtimeCmd := &cobra.Command{
 		Use:   "runtime",
-		Short: "runtime Agent utility commands",
+		Short: "Runtime Security Agent utility commands",
+	}
+
+	confFilePath := ""
+	if len(globalParams.ConfigFilePaths) != 0 {
+		confFilePath = globalParams.ConfigFilePaths[0]
 	}
 
-	runtimeCmd.AddCommand(commonPolicyCommands(globalParams)...)
-	runtimeCmd.AddCommand(selfTestCommands(globalParams)...)
-	runtimeCmd.AddCommand(activityDumpCommands(globalParams)...)
-	runtimeCmd.AddCommand(securityProfileCommands(globalParams)...)
-	runtimeCmd.AddCommand(processCacheCommands(globalParams)...)
-	runtimeCmd.AddCommand(networkNamespaceCommands(globalParams)...)
-	runtimeCmd.AddCommand(discardersCommands(globalParams)...)
+	sysprobeGlobalParams := &sysprobecmd.GlobalParams{
+		ConfFilePath:         confFilePath,
+		FleetPoliciesDirPath: globalParams.FleetPoliciesDirPath,
+	}
+
+	runtimeCmd.AddCommand(policy.Command(sysprobeGlobalParams))
```

Comment:
how does this work related to the logger configuration ?

Reply:
Not well. I'm on it